frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/emoji_picker/src/default_emoji_picker_view.dart

```diff
@@ -278,14 +270,16 @@ class DefaultEmojiPickerViewState extends State<DefaultEmojiPickerView>
       onPressed: () {
         widget.state.onEmojiSelected(categoryEmoji.category, emoji);
       },
-      child: FittedBox(
-        fit: BoxFit.scaleDown,
-        child: Text(
-          emoji.emoji,
-          textScaleFactor: 1.0,
-          style: TextStyle(
-            fontSize: emojiSize,
-            backgroundColor: Colors.transparent,
+      child: FlowyHover(
+        child: FittedBox(
+          fit: BoxFit.scaleDown,
+          child: Text(
+            emoji.emoji,
+            textScaleFactor: 1.0,
```

Comment:
Do we need to force `textScaleFactor: 1.0` here?

Reply:
No, I think it is unnecessary. I also removed the `BoxFit`, the default value `BoxFit.contain` should be enough here.

```dart
   child: FlowyHover(
        child: FittedBox(
          child: Text(
            emoji.emoji,
            style: TextStyle(
              fontSize: emojiSize,
            ),
          ),
        ),
      ),
```